src/app/core/header/header.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="header">
+    <angular-logo></angular-logo>
+    <div class="title">          
+        Video portal
+    </div>
+    <div class="col">
+        <a class="profile">
```

Comment:
We don't need to wrap links to another one. So, I guess, it could be `div`

same for 12 line

Reply:
note: this one already fixed in the next PR (homework 2)